src/exasol_odbc/dialect.tdd

```diff
@@ -230,23 +230,23 @@
       <argument type='str' />
     </function>
     <function group='string' name='MID' return-type='str'>
-      <formula>&#10;(CASE WHEN (%2 IS NULL) OR %2 &lt; 1 THEN NULL&#10;      ELSE {fn SUBSTRING(%1,{fn FLOOR(%2)},LENGTH(%1) - ({fn FLOOR(%2)}) + 1)} END)</formula>
+      <formula>MID(%1, %2)</formula>

```

Comment:
Null handling is missing in online handbook
https://docs.exasol.com/sql_references/functions/alphabeticallistfunctions/mid.htm
Please get that added by the tech writers. Without that I can't judge what Exasol's `MID` is supposed to do in that case and whether special care is necessary.
Also the current implementation uses `FLOOR`, indicating that floating point numbers are acceptable input. Check that `MID` behaves the same way.

Reply:
When one of the values is null, then the function returns null